examples/atari/atari_dqn.py

```diff
@@ -41,6 +41,13 @@ def get_args():
     )
     parser.add_argument('--frames-stack', type=int, default=4)
     parser.add_argument('--resume-path', type=str, default=None)
+    parser.add_argument('--resume-id', type=str, default=None)
+    parser.add_argument(
+        '--logger',
```

Comment:
and maybe we can add some instructions on how to use wandb (including resume) in `examples/atari/README.md` instead of tensorboard?

Reply:
Can we add the instructions in doc also? I can make a separate PR tomorrow for adding the detailed instructions in `examples/atari/README.md` as well as docs.

Reply:
Just append to this pr